### DISCUSSION TITLE
After deleting a file field, the query still returns a URL

### DISCUSSION THREAD

**Discussion by diogomartino** [pocketbase/pocketbase#3158] [Q&A] [answered]
Hi guys, I've been using PocketBase for a few days and I'm having a problem that I'm not sure if it's me not understanding how this works or it's a bug.

In my table I have a record that has `cover` field, which is a file that is supposed to be optional. I query that record and it returns a URL pointing to a file that exists. After I send an update (using `multipart/form-data`) that sets `cover` as an empty string, the file is deleted but when I query that record again, the `cover` returns a partial URL without the filename, eg:` /api/files/qfs83s2dodyp5s9/v8rf6citkvs8195/`. Is this intended? And if so, why? IMO it makes checking if the record as a file or not more complex than it needs to be.

**Comment by diogomartino** [comment, marked as answer]
Nevermind, the thing that's setting that partial URL is `pb.files.getUrl`. I forgot I was using it, although maybe when an empty filename is passed it should not return a partial url but null 🤔

**Reply by ganigeorgiev** [reply]
_I'll update it with the next SDK release to return empty string._
